core/domain/stats_domain.py

```diff
@@ -41,6 +41,158 @@
 MIGRATED_STATE_ANSWER_TIME_SPENT_IN_SEC = 0.0
 
 
+class ExplorationStats(object):
+    """Domain object representing analytics data for an exploration."""
+
+    def __init__(self, exp_id, exp_version, num_actual_starts, num_completions,
+                 state_stats_mapping):
+        """Constructs an ExplorationStats domain object.
+
+        Args:
+            exp_id: str. ID of the exploration.
+            exp_version: int. Version of the exploration.
+            num_actual_starts: int. Number of learners who actually attempted
```

Comment:
should we explain meaning of actually here as well?

Reply:
Done.